h/search/query.py

```diff
@@ -251,56 +223,34 @@ class DeletedFilter(object):
     deleted.
     """
 
-    def __call__(self, _):
-        return {"bool": {"must_not": {"exists": {"field": "deleted"}}}}
+    def __call__(self, search, _):
+        return search.exclude("exists", field="deleted")
 
 
 class NipsaFilter(object):
+    """Return an Elasticsearch filter for filtering out NIPSA'd annotations."""
```

Comment:
This documentation is slightly incorrect in that `__call__` doesn't return an Elasticsearch filter. I think you could reword this to say simply "Filter out hidden (NIPSA'd) annotations".